gapic/src/main/com/google/gapid/perfetto/models/CounterTrack.java

```diff
@@ -136,19 +145,21 @@ private String counterSQL() {
         return Data.empty(null);
       }
 
-      Data data = new Data(null, new long[rows + 1], new double[rows + 1]);
+      Data data = new Data(null, new long[rows + 1], new double[rows + 1], new long[rows + 1]);
       res.forEachRow((i, r) -> {
         data.ts[i] = r.getLong(0);
         data.values[i] = r.getDouble(2);
+        data.ids[i] = r.getLong(3);
```

Comment:
Can we remove the (1) element from the rangeSql if its not getting used anywhere?

Reply:
It's actually used at `data.ts[rows] = res.getLong(rows - 1, 1, 0);`.